docs/rfc/grouping-api.md

```diff
@@ -0,0 +1,55 @@
+# Grouping API
+
+We will be adding the ability to add 'grouping' support to `react-beautiful-dnd`. I am attempting to create naming for the new api that feels right. The grouping feature will enable ability to hover over a Draggable to group with it. Right now I have been calling the feature 'grouping'. However, I am not sure if even that is the right name for the feature.
+
+What the consumer actually achieves with this feature may vary. By default when dropping a item over another one, the dragging item will drop _into_ the one being hovered over
+
+## API
+
+### `Draggable` > isGroupingEnabled
```

Comment:
Right now it is actually only: `Draggable` > ~~`isGroupingEnabled`~~ `isCombineEnabled`

Which means that it is opt in / out for the whole group. This is easier, but less flexible. I am guessing there are lists where there are some items that can be combined with and some items that cannot. 

I was thinking of keeping it simple and just having it as opt in / opt for the list.

Thoughts?